### PR TITLE
test setup for ssl

### PR DESCRIPTION
**Pull request by mbukatov**
Work in progress, remaining items:

* [x] variable for disabling port 80
* [x] add redirect setup
* [x] add playbook for setup using QE CA
* [x] resolve comments from others (so far)
* [x] make sure QE CA is installed on qe server and client (for us to be able to receive alerts via https)
* [x] account for problem with custom CA use case

Based on https://github.com/Tendrl/documentation/wiki/Enabling-Https-on-tendrl-server, some code reused from https://github.com/Tendrl/tendrl-ansible/pull/46/files.

### CODE REVIEW

File: roles/tendrl-httpd-ssl/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+
+- name: restart httpd
+  service:
+    name=httpd
+    state=restarted
```
Comment (dahorak) [inline roles/tendrl-httpd-ssl/handlers/main.yml]: Do we need full restart of `http` service? Wouldn't be enough `state: reloaded`?

File: roles/tendrl-httpd-ssl/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+
+- debug:
+    var: httpd_server_name
+
+- debug:
+    var: httpd_ip_address
```
Comment (dahorak) [inline roles/tendrl-httpd-ssl/tasks/main.yml]: Would it make sense, to use some defaults for `httpd_server_name` and for `httpd_ip_address`?

File: test_setup.tendrl_httpd_ssl_selfsigned.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+#  Setup of HTTPS for Tendrl UI, API and Grafana
+# ===============================================
+#
+# This is **minimal setup**, which **uses default local ssl certs** provided by
+# mod_ssl rpm package.
+
+- hosts: usm_server
+  remote_user: root
+  handlers:
+    - name: restart httpd
```
Comment (dahorak) [inline test_setup.tendrl_httpd_ssl_selfsigned.yml]: The handler is already defined in the role, isn't it? Or did I miss something?

Comment (mbukatov) [reply test_setup.tendrl_httpd_ssl_selfsigned.yml]: good point

File: roles/tendrl-httpd-ssl/handlers/main.yml (hunk 1 above)
Comment (mbukatov) [reply roles/tendrl-httpd-ssl/handlers/main.yml]: good point, fixed: 7bbe9c0c78743a19dd5e2c814947afc5e0cddbcb

File: roles/tendrl-httpd-ssl/tasks/main.yml (hunk 2 above)
Comment (mbukatov) [reply roles/tendrl-httpd-ssl/tasks/main.yml]: Actually, yes.

**Comment by mbukatov** [comment]
@dahorak the pull request is complete, and includes 2 fixes wrt original steps on the wiki

**Comment by mbukatov** [comment]
@ebondare when the final draft of the ssl docs is available, please check that this pull request matches it

**Comment by dahorak** [comment]
Merging in this state, will create new PR if necessary once documentation for SSL configuration will be finalized.

**Pull request merged**